go.mod

```diff
@@ -4,7 +4,7 @@ go 1.21
 
 require (
 	github.com/aws/aws-sdk-go v1.44.122
-	github.com/scaleway/scaleway-sdk-go v1.0.0-beta.12
+	github.com/scaleway/scaleway-sdk-go v1.0.0-beta.21
```

Comment:
Are there any breaking changes from beta.12 to beta.21 we need to watch out for?

Reply:
We don"t have BC when upgrade this deps. I have check other tables.